cluster/config-defaults.yaml

```diff
@@ -280,6 +280,13 @@ fabric_gateway_controller_memory: "1Gi"
 fabric_gateway_crd_v1_enabled: "false"
 
 # kube-api-server settings
+
+# EventRateLimit admission plugin configuration
+# Reference: https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#eventratelimit
+event_rate_limit_enable: "true"
+event_rate_limit_config_qps: 500
+event_rate_limit_config_burst: 1000
```

Comment:
Similar to https://github.com/zalando-incubator/kubernetes-on-aws/pull/5466/files#r1001858221 let's make these strings

Reply:
Alright!

Reply:
Done. Thanks!